<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkRegionGrowingControls</class>
 <widget class="QWidget" name="QmitkRegionGrowingControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>541</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkRegionGrowing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_SelectedImageLabel">
     <property name="text">
      <string>region grower</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkPointListWidget" name="m_PointListWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="m_ButtonRegionGrow">
     <property name="toolTip">
      <string>start dumb segmentation</string>
     </property>
     <property name="text">
      <string>Start region grower</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkPointListWidget</class>
   <extends>QListView</extends>
   <header>QmitkPointListWidget.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">QmitkDataStorageComboBox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
